dotCMS/src/integration-test/java/com/dotcms/concurrent/DotConcurrentFactoryTest.java

```diff
@@ -10,6 +10,79 @@
 
 public class DotConcurrentFactoryTest {
 
+    @Test
+    public void testDefaultOne_Submitter_Conifg() throws JSONException{
+
+        final String submitterName = "testsubmitter";
+        final DotConcurrentFactory dotConcurrentFactory =
+                DotConcurrentFactory.getInstance();
+
+        final DotSubmitter submitter =
+                dotConcurrentFactory.getSubmitter(submitterName,
+                        new DotConcurrentFactory.SubmitterConfigCreatorBuilder().poolSize(2)
+                        .maxPoolSize(4).queueCapacity(500).build()
+                );
+
+        System.out.println(submitter);
+
+        IntStream.range(0, 40).forEach(
+                n -> {
+
+                    if (n % 10 == 0) {
+
+                        System.out.println(submitter);
+                    }
+                    submitter.execute(new PrintTask("Thread" + n));
+                }
+        );
+
+        //check active thread, if zero then shut down the thread pool
+        for (;;) {
+            int count = submitter.getActiveCount();
+            System.out.println("Active Threads : " + count);
```

Comment:
Codacy found an issue: [The String literal "Active Threads : " appears 4 times in this file; the first occurrence is on line 42](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6265363)